packages/vega-typings/types/runtime/dataflow.d.ts

```diff
@@ -0,0 +1,28 @@
+export interface Changeset {
+  insert(tuples: any): this;
+  remove(tuples: any): this;
+  modify(tuple: any, field?: string, value?: any): this;
+}
+
+export function changeset(): Changeset;
+
+export class Operator {
+  constructor(init?: any, update?: (obj: any, pulse: any) => any, params?: any, react?: boolean);
+  targets(): any;
+  set(value: any): 1 | 0;
+  skip(): (state: any) => any;
+  modified(): (state: any) => any;
+  parameters(params: object, react?: boolean, initonly?: boolean): Operator[];
+  marshall(stamp: number): any;
+  evaluate(pulse: any): any;
```

Comment:
Do all of these need to return any?

Reply:
Some of these methods have multiple return types:
- skip: boolean (if called with no arguments) | Operator (if called with a state argument)
- modified: boolean (if called with no arguments) | Operator (if called with a state argument)
- evaluate: undefined | StopPropagation dummy object

Others have a singular return type:
- targets: UniqueList object
- marshall: Parameters object

Reply:
Due to time constraints, I'm being opportunistic about updating the typings. Once we flesh out typings for `Tuple` and `Pulse`, I think we can come back and type these to completion.

Reply:
Due to time constraints, I'm being opportunistic about updating the typings. Once we flesh out typings for Tuple and Pulse, I think we can come back and type these to completion. I'd advocate for merging these now, so that downstream projects can more immediately reap the benefits.

Reply:
Sounds good to me. 

If you know that there will be some type later, you can always create an alias. 

For example:

```ts
type Tuple = any;
```